scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
Well, I think the lack of s390x support tells the story quite well: cross building is already supported by the go tool chain and yet we don't have it.